arduino-ide-extension/src/node/boards-service-impl.ts

```diff
@@ -199,8 +201,28 @@ export class BoardsServiceImpl
     const req = new BoardSearchRequest();
     req.setSearchArgs(query || '');
     req.setInstance(instance);
+    return this.handleListBoards(client.boardSearch.bind(client), req);
+  }
+
+  async getInstalledBoards(): Promise<BoardWithPackage[]> {
+    const { instance, client } = await this.coreClient;
+    const req = new BoardListAllRequest();
+    req.setInstance(instance);
+    return this.handleListBoards(client.boardListAll.bind(client), req);
+  }
+
+  async handleListBoards(
```

Comment:
The visibility should be `private` if it's not called from outside of the module.